src/SS58Address.h

```diff
@@ -0,0 +1,92 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Base58.h"
+#include "Data.h"
+#include "PublicKey.h"
+
+#include <array>
+#include <string>
+#include <iostream>
+
+const std::string SS58Prefix = "SS58PRE";
+
+namespace TW {
+
+class SS58Address {
```

Comment:
What is 'SS58'stands for?  Could you add a comment?

Reply:
SS58 is short for **S**ub**S**trate Base**58** checksum, we have a link in `TWSS58AddressType.h`